pkg/util/flags/flags.go

```diff
@@ -72,3 +73,22 @@ func SetFlagsFromEnv(flags *pflag.FlagSet, prefix string) error {
 
 	return nil
 }
+
+// GetFlagsAndValues returns all flags and their values as a slice with elements in the format of
+// "--<flag>=<value>"
+func GetFlagsAndValues(flags *pflag.FlagSet, excludeFilter string) []string {
```

Comment:
should the excludeFilter be of type []string in order to include more filters? Or even better, a regex pattern?